### ISSUE TITLE
Problems in installation

### ISSUE THREAD

**Issue by syyuan1993** [Zardinality/TF_Deformable_Net#20]
Thanks a lot for the great lib! I'm trying to follow your work, yet facing some problems. When I tried to compile, an error notice is shown as following:
/home/siyangyuan/anaconda3/envs/tf14/lib/python3.5/site-packages/tensorflow/include
/home/siyangyuan/anaconda3/envs/tf14/lib/python3.5/site-packages/tensorflow/include/unsupported/Eigen/CXX11/../../../Eigen/src/Core/arch/CUDA/Half.h(390): error: identifier "hexp" is undefined

/home/siyangyuan/anaconda3/envs/tf14/lib/python3.5/site-packages/tensorflow/include/unsupported/Eigen/CXX11/../../../Eigen/src/Core/arch/CUDA/Half.h(400): error: the global scope has no "hlog"

/home/siyangyuan/anaconda3/envs/tf14/lib/python3.5/site-packages/tensorflow/include/unsupported/Eigen/CXX11/../../../Eigen/src/Core/arch/CUDA/Half.h(413): error: identifier "hsqrt" is undefined

/home/siyangyuan/anaconda3/envs/tf14/lib/python3.5/site-packages/tensorflow/include/unsupported/Eigen/CXX11/../../../Eigen/src/Core/arch/CUDA/Half.h(435): error: identifier "hfloor" is undefined

/home/siyangyuan/anaconda3/envs/tf14/lib/python3.5/site-packages/tensorflow/include/unsupported/Eigen/CXX11/../../../Eigen/src/Core/arch/CUDA/Half.h(442): error: identifier "hceil" is undefined

/home/siyangyuan/anaconda3/envs/tf14/lib/python3.5/site-packages/tensorflow/include/unsupported/Eigen/CXX11/../../../Eigen/src/Core/arch/CUDA/Half.h(530): error: the global scope has no "hlog"

/home/siyangyuan/anaconda3/envs/tf14/lib/python3.5/site-packages/tensorflow/include/unsupported/Eigen/CXX11/../../../Eigen/src/Core/arch/CUDA/PacketMathHalf.h(291): error: identifier "h2log" is undefined

/home/siyangyuan/anaconda3/envs/tf14/lib/python3.5/site-packages/tensorflow/include/unsupported/Eigen/CXX11/../../../Eigen/src/Core/arch/CUDA/PacketMathHalf.h(296): error: identifier "h2exp" is undefined

/home/siyangyuan/anaconda3/envs/tf14/lib/python3.5/site-packages/tensorflow/include/unsupported/Eigen/CXX11/../../../Eigen/src/Core/arch/CUDA/PacketMathHalf.h(301): error: identifier "h2sqrt" is undefined

/home/siyangyuan/anaconda3/envs/tf14/lib/python3.5/site-packages/tensorflow/include/unsupported/Eigen/CXX11/../../../Eigen/src/Core/arch/CUDA/PacketMathHalf.h(306): error: identifier "h2rsqrt" is undefined

10 errors detected in the compilation of "/tmp/tmpxft_00003e41_00000000-7_deform_conv.cu.cpp1.ii".

Which is very confusing. Do you have any thoughts about what might cause this? I'm using gcc 4.9.3, tensorflow 1.4 installed with binary file, cuda 8.0, cudnn 6, Titan X. Thanks a lot!

**Comment by Zardinality** [comment]
Sorry, I don't have a clue about what is going on here. You seem to be the first that encounter this error.

**Comment by syyuan1993** [comment]
@Zardinality Thanks anyway. It happens not only here but also when I tried to build the code of Tf_Deformable_conv repository. Any more suggestions would be appreciated.

**Comment by ghost** [comment]
@syyuan1993    Did you solve this problem?  can you give some suggestions?

**Comment by syyuan1993** [comment]
> @syyuan1993 Did you solve this problem? can you give some suggestions?

Nope, eventually, I used a pytorch implementation of deformable net.

**Comment by ghost** [comment]
@syyuan1993   Thank you very much .

**Comment by cristinasegalin** [comment]
has this been solved? I have the same problem
